### PR TITLE
docs(Java): Add `application.yaml` filename to YAML code snippets

### PR DESCRIPTION
**Pull request by bugwelle**
All of these snippets are for `application.yaml`, but often, it wasn't mentioned anywhere.  By adding the filename, it becomes more clear where the snippet is supposed to go.

### CODE REVIEW

File: guides/databases-sqlite.md [hunk 1]
```diff
@@ -232,7 +232,7 @@ With that in place, the server will use this prepared database instead of bootst
 
 Finally, configure the DB connection - ideally in a dedicated `sqlite` profile:
 
-```yaml
+```yaml [srv/src/main/resources/application.yaml]
```
Comment (renejeglinsky) [inline guides/databases-sqlite.md:235]: This only works when using `::: code-group` as shown in line 147

Comment (bugwelle) [reply guides/databases-sqlite.md:235]: Ah, nevermind. I will adapt all of my changes. :)

Comment (bugwelle) [reply guides/databases-sqlite.md:236]: Fixed

File: java/cqn-services/persistence-services.md [hunk 2]
```diff
@@ -192,7 +192,8 @@ Automatic schema deployment isn't suitable for productive use. Consider using pr
 
 If you don't have a compatible PostgreSQL service binding in your application environment, you can also explicitly configure the connection data of your PostgreSQL database in the _application.yaml_:
 
-```yaml
+::: code-group
```
[the remark this answers is not in the document]
Comment (bugwelle) [reply java/cqn-services/persistence-services.md:195]: @renejeglinsky Did I miss something? This was copy&paste and is the only place that's commented on.

Locally, it looks correct:

Screenshot 2024-06-18 at 16 27 29

[the remark this answers is not in the document]
Comment (renejeglinsky) [reply java/cqn-services/persistence-services.md:195]: It looks fine for me. Let's merge and see if that works then as well.

**Review by renejeglinsky — APPROVED** [review]

**Comment by bugwelle** [comment]
This PR was superseded by #1059

**Pull request closed without merging**